### PR TITLE
[Refactor] Pasteboard

### PR DESCRIPTION
**Pull request by rakaramos**
## Summary
- By checking the [documentation](https://developer.apple.com/documentation/appkit/nspasteboard/pasteboardtype/1527185-string), it seems that we can our `extension` on `NSPasteboard.PasteboardType`, because the default `.string` is already what we want.
```swift
NSPasteboard.PasteboardType.string.rawValue // prints "public.utf8-plain-text"
```
- The rest of the refactor is just moving from imperative to declarative.

## Review Checklist
- [ ] I have responded to reviews and made changes where appropriate.
- [x] I have tested the code
- [ ] I have updated comments / documentation related to the changes I made.
- [x] I have rebased my PR branch onto xi-mac/master.

### CODE REVIEW

File: Sources/XiEditor/EditViewController.swift [hunk 1]
```diff
@@ -559,15 +555,11 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
     }
 
     @objc func paste(_ sender: AnyObject?) {
-        let pasteboard = NSPasteboard.general
-        if let items = pasteboard.pasteboardItems {
-            for element in items {
-                if let str = element.string(forType: .PlainText) {
-                    document.sendPaste(str)
-                    break
```
[the remark this answers is not in the document]
Comment (rakaramos) [reply Sources/XiEditor/EditViewController.swift]: yeah, IMHO we should not stop, but I could be wrong

[the remark this answers is not in the document]
Comment (rakaramos) [reply Sources/XiEditor/EditViewController.swift]: What's more, the API exposes [an array of pasteboard items](https://developer.apple.com/documentation/appkit/nspasteboard/1529995-pasteboarditems), not a set, so we have no guarantees that there will be only one `string` value on it.

[the remark this answers is not in the document]
Comment (rakaramos) [reply Sources/XiEditor/EditViewController.swift]: @cmyr I've changed to use `string` method on `NSPasteboard`
[here](https://developer.apple.com/documentation/appkit/nspasteboard/1533566-string) are the docs

[the remark this answers is not in the document]
Comment (cmyr) [reply Sources/XiEditor/EditViewController.swift]: Okay that looks/sounds right to me. 👍

**Pull request merged**